package.json

```diff
@@ -8,13 +8,13 @@
     "i18next-client": "^1.11.5",
     "jasmine": "^3.0.0",
     "jshint": "~2.11.0",
-    "opening_hours": "~3.5.0"
+    "simple-opening-hours": "^0.1.1"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/wo-ist-markt/wo-ist-markt.github.io"
   },
- "scripts": {
+  "scripts": {
```

Comment:
This new whitespace should be removed.

Reply:
@Kishore-abhimanyu Do you need help with this?

Reply:
@Kishore-abhimanyu Are you still working on this?

Reply:
@mojoaxel Do you know the state of this pull request?

Reply:
:bellhop_bell: @Kishore-abhimanyu, @mojoaxel Last reminder before closing this.